pyuvsim/tests/test_run.py

```diff
@@ -49,6 +49,15 @@ def test_run_paramfile_uvsim(goto_tempdir, paramfile):
     uv_ref.read_uvfits(os.path.join(SIM_DATA_PATH, 'testfile_singlesource.uvfits'))
     uv_ref.unphase_to_drift(use_ant_pos=True)
     uv_ref.reorder_blts("time", minor_order="baseline")
+    # This is an old file with the bug that added one to the
+    # antenna numbers for uvfits files. Fix them (if pyuvdata is recent)
+    if np.min(np.union1d(uv_ref.ant_1_array, uv_ref.ant_2_array)) > 0:
```

Comment:
Is the uvfits bug the _only_ way this can happen though? ant_1_array and ant_2_array should be indices correct? So the answer to my first question should be yes?

Reply:
They're not indices, they're antenna numbers. But until recently we were subtracting 1 from them (as well as from the antenna_numbers) when we wrote out uvfits files. That and a bug in our baseline number calculation was fixed in: https://github.com/RadioAstronomySoftwareGroup/pyuvdata/pull/1148

Reply:
Yeah I remember that, just wondering if the check posted here will only trigger off that bug right? or are there other ways for this to get triggered even if we didn't want it?

Reply:
I would not use this generically, it's tuned for the particular file used in this test.

Reply:
Antenna numbers can generally be zero, but this file is made from an antenna layout csv that has antenna numbers starting at 1.

Reply:
Ah yeah great okay. Tests. I must be distracted.